docker/openemr/5.0.2/Dockerfile

```
@@ -50,7 +50,10 @@ COPY openemr.conf /etc/apache2/conf.d/
 COPY run_openemr.sh autoconfig.sh auto_configure.php /var/www/localhost/htdocs/openemr/
 COPY utilities/unlock_admin.php utilities/unlock_admin.sh /root/
 RUN chmod 500 run_openemr.sh autoconfig.sh /root/unlock_admin.sh \
-    && chmod 000 auto_configure.php /root/unlock_admin.php
+    && chmod 000 auto_configure.php /root/unlock_admin.php \
+    && rm -rf /var/www/localhost/cgi-bin \
+    && rm -f /var/www/localhost/htdocs/index.html \
+    && chown -R /var/www/localhost apache:apache
```

the above chown line raises a subconscious red flag for me for some reason. let me look into that a bit.

---

What might be setting off the red flag is that the syntax is backwards on that `chown`, should be as below to have the intended effect. That said, definitely agreed that it's a lot to assume apache should own everything.
```suggestion
    && chown -R apache:apache /var/www/localhost 
```
(Yes I'm still alive! Procrastinating studying for a cardio test on Tuesday!)